File: aulas/2026-08-17/02-calculo-com-potencia/java/Main.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Main {
    public static void main(String[] args) {
        double venda= 3.2;
        double taxa= 1.3;
        int meses = 3;
        double resultado = venda * Math.pow(taxa,meses);
        System.out.println("O resultado é " + resultado);
    }
}

File: aulas/2026-08-17/02-calculo-com-potencia/cpp/main.cpp
#include <iostream>
#include <math.h>

int main() {
    float venda= 3.2;
    float taxa= 1.3;
    int meses = 3;
    float resultado = venda * pow(taxa,meses);
    std::cout << "O resultado é " << resultado << std::endl;
    return 0;
}